### ISSUE TITLE
Cellassign tutorial is not working due to gdrive permissions

### ISSUE THREAD

**Issue by ori-kron-wis** [scverse/scvi-tutorials#355]
Cellassign tutorial was used to work on Berkeley workstations, but now from WIS it has a permission error to access gdrive to fetch the data.
One possible solution is to re create those files:

To demonstrate CellAssign, we use the data from the original publication, which we converted into h5ad format. The data are originally available from here:

https://zenodo.org/record/3372746

```pytb
File /usr/local/lib/python3.12/site-packages/gdown/download.py:278, in download(url, output, quiet, proxy, speed, use_cookies, verify, id, fuzzy, resume, format, user_agent, log_messages)
    268     except FileURLRetrievalError as e:
    269         message = (
    270             "Failed to retrieve file url:\n\n{}\n\n"
    271             "You may still be able to access the file from the browser:"
   (...)
    276             url_origin,
    277         )
--> 278         raise FileURLRetrievalError(message)
    280 filename_from_url = None
    281 last_modified_time = None

FileURLRetrievalError: Failed to retrieve file url:

	Cannot retrieve the public link of the file. You may need to change
	the permission to 'Anyone with the link', or have had many accesses.
	Check FAQ in https://github.com/wkentaro/gdown?tab=readme-ov-file#faq.

You may still be able to access the file from the browser:

	https://drive.google.com/uc?id=10l6m2KKKioCZnQlRHomheappHh-jTFmx

but Gdown can't. Please check connections and permissions.

```

#### Versions:

1.2

**Comment by ori-kron-wis** [comment]
The original files in Zenodo (https://zenodo.org/record/3372746) are in R and we need to convert them to h5ad somehow. I tried different methods but none worked (zellkonverter <-> SingleCellExperiment). Also, I was looking for those temp files downloaded from gdrive in s133 during the run, but no luck either. I also tried to run the tutorial myself on s133 and gdrive was restricted again but this was from my user, I couldn't do it from Martin's (not sure it will help either).

So we kinda stuck with no files for this tutorial.
